extensions/positron-python/python_files/positron/positron_ipykernel/ui.py

```diff
@@ -81,6 +93,7 @@ def _set_console_width(params: List[JsonData]) -> None:
 
 _RPC_METHODS: Dict[str, Callable[[List[JsonData]], JsonData]] = {
```

Comment:
```suggestion
_RPC_METHODS: Dict[str, Callable[["PositronIPyKernel", List[JsonData]], Optional[JsonData]]] = {
```

Reply:
This should address the pyright 1.1.308 based type checking failures in CI

Reply:
oh thanks! Was having some trouble getting this check to run locally yesterday.